public/stylesheets/site/2.0/21-userstuff.css

```
@@ -119,8 +119,8 @@
 .userstuff caption {
   font-size: 1em;
   height: auto;
-  opacity: 1.0;
   width: auto;
+      opacity: 1;
```

```suggestion
    opacity: 1;
```

(If you compare it to `word-wrap` on line 4, you can see `opacity` is currently indented a total of 6 spaces instead of the desired 4.)